DAQ/src/CrvDigisFromFragments_module.cc

```diff
@@ -64,16 +64,16 @@ CrvDigisFromFragments::CrvDigisFromFragments(fhicl::ParameterSet const& pset)
   , diagLevel_(pset.get<int>("diagLevel",0))
```

Comment:
Why is this class in the art namespace and not the mu2e namespace?  Same question for StrawAndCaloDigisFromFragments_module.cc .  I did not check the others.

Reply:
I have no clue. I did that because Tomo was doing it. I'm pretty sure the code should work also if I use the mu2e namespace. Probably he was doing that for debugging purposes?